pavelib/assets.py

```diff
@@ -21,13 +22,19 @@
 COFFEE_DIRS = ['lms', 'cms', 'common']
 # A list of directories.  Each will be paired with a sibling /css directory.
 SASS_DIRS = [
-    path("lms/static/sass"),
+#    path("lms/static/sass"),
     path("lms/static/themed_sass"),
     path("cms/static/sass"),
     path("common/static/sass"),
     path("lms/static/certificates/sass"),
 ]
-SASS_LOAD_PATHS = ['common/static', 'common/static/sass']
+SASS_FILES_AND_TARGETS = [
```

Comment:
The explicit file load paths will go away, once I kill the redundant scss files (it's really only there now so I can get some timing information), and we'll go back to doing the whole of `lms/static/sass` in one shot. That being said, isn't the open certs stuff covered by `lms/static/certificates/sass` above? With the exception of `lms/static/sass`, I haven't messed with the load paths, so if it's not here, it might be broken on master.